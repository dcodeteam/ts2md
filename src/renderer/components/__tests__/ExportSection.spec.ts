import { testRenderer } from "../../../__tests__/test-utils";

testRenderer("export");
