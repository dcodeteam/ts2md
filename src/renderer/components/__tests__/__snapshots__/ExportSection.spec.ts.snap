// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`export export-reexport.ts 1`] = `
### Class \`ClassConstructor\`

This is ClassConstructor documentation.

\`\`\`javascript
const instance = new ClassConstructor("foo", 1);
\`\`\`

#### Constructors

Creates new instance.

\`\`\`typescript
class ClassConstructor {
  public constructor(foo: string, bar: number);
}
\`\`\`

###### Parameters

- **foo**: \`string\`
  This is \`foo\` argument.
- **bar**: \`number\`
  And this is \`bar\` argument.

`;

exports[`export export-reexport-ordering.ts 1`] = `
### Variable \`FOO\`

Represents value of \`FOO\`.

\`\`\`typescript
const FOO: "foo";
\`\`\`

### Function \`functionArguments\`

This is function docs.

\`\`\`typescript
function functionArguments(a: number, b: { c: number }): number;
\`\`\`

###### Parameters

- **a**: \`number\`
  this is parameter a docs
  this is parameter \`a\`.
- **b**: \`{ c: number; }\`
  this is parameter b docs
  this is parameter \`b\`.

###### Returns

- \`number\`

### Class \`ClassConstructor\`

This is ClassConstructor documentation.

\`\`\`javascript
const instance = new ClassConstructor("foo", 1);
\`\`\`

#### Constructors

Creates new instance.

\`\`\`typescript
class ClassConstructor {
  public constructor(foo: string, bar: number);
}
\`\`\`

###### Parameters

- **foo**: \`string\`
  This is \`foo\` argument.
- **bar**: \`number\`
  And this is \`bar\` argument.

`;
